config/ci-tests/verify_cuda_n2k_pl1bitcorr.yaml

```diff
@@ -162,3 +162,4 @@ check_data:
     first_buf: host_counts_buffer
     second_buf: simulated_counts_buffer
     num_frames_to_test: buffer_depth
+    check_metadata: True
```

Comment:
Out of scope for this PR, but we should have an example config file that includes comments/docs for available stages, options, etc.